packages/helia/.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
```

Comment:
I added helia as a package which is NOT ideal but it was the fastest way to get it integrated, from my perspective.  We should ultimately update it to be a submodule like lfa but I don't have it in me to do it right now.